block/block_service.proto

```diff
@@ -70,43 +70,45 @@ message PublishStreamRequest {
         /**
          * An item sent when a stream is ended by the publisher.
          */
-        EndOfStream status = 2;
+        // @workaround(PBJ #280) REMOVED pending fix for PBJ bug 280
```

Comment:
We will enable this in later PR ?

Reply:
Yes, once the PBJ bug (as noted) is fixed (it's been waiting on resources since June), we'll add this back in.  Hopefully within a month or two (it's not a trivial bug, it requires some pretty large improvements to PBJ's code generation).